### ISSUE TITLE
Undocked playback controls disappear when scaled and inconsistencies

### ISSUE THREAD

**Issue by dotdivee** [musescore/MuseScore#17301]
### Issue type

UI bug

### Bug description

It seems that there is no minimum width/height provided for the playback controls yet they are resizable when undocked. This means that their width and/or height can be scaled to 0 rendering them inaccessible as they are made invisible and cannot be redocked.

This is problematic as the only way to reset this is to go to ```View>Toolbars>Playback Controls``` and toggle and un-toggle the playback controls to reset them to a docked position.

Furthermore I found that while scaling the UI vertically the controls are aligned centrally yet when scaled horizontally they are justified from the left rather than centrally and the empty space around controls cannot be used to drag the playback control window around.

### Steps to reproduce

1. Undock the playback controls in a score
2. Move cursor to edge of playback control window to make scale controls appear
3. Drag cursor until width and/or height is reduced to 0

### Screenshots/Screen recordings

# Controls disappearing
![disappearing](https://user-images.githubusercontent.com/69803052/233492254-6a94a07d-781e-4c96-9613-030d3bd068a2.png)

# Alignment issue
![align](https://user-images.githubusercontent.com/69803052/233492095-8dde36fc-74ea-4745-b09b-271bbb528d96.png)

### MuseScore Version

MuseScore 4.0.2-230651553

### Regression

I don't know

### Operating system

Windows 10

**Comment by cbjeukendrup** [comment]
The problem here is that those toolbars should not be resizable at all. On other operating systems, they aren't indeed, but on Windows specifically, they are.

**Comment by dotdivee** [comment]
Yeah I have to agree. I suppose there is really no reason as to why they should be resizable.
